package.json

```diff
@@ -2,7 +2,9 @@
   "name": "balena-pine",
   "version": "11.0.2",
   "description": "Balena PineJS client.",
-  "main": "build/pine.js",
+  "main": "es5/pine.js",
+  "types": "es5/pine.d.ts",
+  "browser": "es5/pine.js",
```

Comment:
Afaik if you don't specify this, it will just default to the `main` field.
But I see that you might want this for the follow-up PR.

Reply:
Yeah I went for being explicit and it will be different in the later PR